.eslintrc.cjs

```diff
@@ -18,6 +18,7 @@ module.exports = {
 		},
 		{
 			extends: [
+				"plugin:jsdoc/recommended-typescript",
```

Comment:
I can't comment on `src/greet.ts` because it's not part of the files changed - but this change introduced a complaint there. You can see it in the file viewer on GitHub and by running `pnpm lint` locally.

```plaintext
/home/runner/work/template-typescript-node-package/template-typescript-node-package/src/greet.ts
Warning:   5:8  warning  Missing JSDoc comment  jsdoc/require-jsdoc
```

I don't think we want to enforce that _every_ function has a JSDoc comment. So I think we should turn this rule off.

Reply:
I added ```/* eslint-disable require-jsdoc */``` to ```src/greet.ts``` but it is still failing the same checks, so I have not yet found the solution. I am not exactly sure what the problem is, that is why I requested the review again earlier.

Reply:
> `/* eslint-disable require-doc */`

The rule's reporting under `jsdoc/require-jsdoc`, so you would need to disable that name. But I think we'd want to disable the rule in _all_ files, in case the user writes more code than just this one. You'll want to read up on https://eslint.org/docs/latest/use/configure/rules.

I also think we'll want all four of these `"off"`s:
https://github.com/JoshuaKGoldberg/ts-api-utils/blob/6050cfe20dcd07ac37b03d69c38801d1ce95ccb7/.eslintrc.cjs#L124-L127

> that is why I requested the review again earlier.

Makes sense! As a general piece of advice for PRs, I'd recommend being very explicit with what you're asking about. Post a comment in the PR with the question and include what you've tried along with what first-party (so here, on eslint.org) docs you've read.